datafusion/physical-expr/src/equivalence/ordering.rs

```diff
@@ -36,15 +36,15 @@ use arrow_schema::SortOptions;
 ///
 /// Here, both `vec![a ASC, b ASC]` and `vec![c DESC, d ASC]` describe the table
 /// ordering. In this case, we say that these orderings are equivalent.
-#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Default)]
```

Comment:
this is a small drive by cleanup -- I can move it to its own PR if preferred

Reply:
I think there is no need to separate.